### PR TITLE
feat: add field settings to module

### PR DESCRIPTION
**Pull request by sebastianthulin**

### CODE REVIEW

File: source/php/Modules/FrontendForm/FrontendForm.php [hunk 1]
```diff
@@ -19,18 +19,23 @@ class FrontendForm extends \Modularity\Module
     public $slug = 'event-form';
     public $supports = [];
 
+    // The field groups that should be displayed in the form.
+    private $fieldGroups = [
+      'group_65a115157a046'
```
Comment (thorbrink) [inline source/php/Modules/FrontendForm/FrontendForm.php:24]: Can we make this less coupled directly to the acf settings? Through passing constructor params maybe?

File: source/php/FieldSettingHidePublic.php
Comment (thorbrink) [inline source/php/FieldSettingHidePublic.php:1]: I would really like to see some tests for this functionality.

Comment (thorbrink) [inline source/php/FieldSettingHidePublic.php:1]: I would really like to see some tests for this functionality.

File: source/php/Modules/FrontendForm/FrontendForm.php (hunk 1 above)
Comment (thorbrink) [inline source/php/Modules/FrontendForm/FrontendForm.php:24]: Can we make this less coupled directly to the acf settings? Through passing constructor params maybe?

**Review by thorbrink — CHANGES_REQUESTED** [review]

File: source/php/Modules/FrontendForm/FrontendForm.php (hunk 1 above)
Comment (sebastianthulin) [reply source/php/Modules/FrontendForm/FrontendForm.php:30]: Conclusion: Do not resolve.

**Review by thorbrink — APPROVED** [review]

**Pull request merged**